app/javascript/actions/messages.js

```diff
@@ -35,14 +36,34 @@ export const fetchMessages = () => (dispatch : T.Dispatch, getState : T.GetState
             .catch(error => dispatch(_messagesReceiveError()))
 }
 
-export const _answerMessage = (messageId : number, answer : string) : T.HumanOverrideMessageAction => ({
-  type: ANSWER,
+export const _answerMessageSuccess = (messageId : number, answer : string) : T.HumanOverrideMessageAction => ({
```

Comment:
Maybe the problem with flow is due to having this messageId typed as integer. Wether is easy to change this type or not, or if it should be an integer or a string, is an entirely different question. Let's pair this tomorrow.